docs/install.rst

```diff
@@ -5,93 +5,50 @@
-  the container as a shared volume. This supports development with a running instance
-  of the service.
-* *master*:  The container contains the master branch of the code at the time the
-  image is built.
-* *release*:  The container contains a release of the code. There will be a
-  separate image for each release.
-
-For more information, see :doc:`docker`.
+The following will describe how to setup an instance of SFM that uses the latest release
+(and is suitable for a production deployment.) See the development documentation for other
+SFM configurations.
 
 SFM *can* be deployed without Docker. The various ``Dockerfile`` s should provide
 reasonable guidance on how to accomplish this.
 
-.. _install-configuration:
-
--------------
-Configuration
--------------
-
-* Passwords are kept in ``secrets.env``.  A template for this file (``example.secrets.env``) is provided.
-* Application credentials for social media APIs are configured in ``secrets.env`` by providing the ``TWITTER_CONSUMER_KEY``,
-  ``TWITTER_CONSUMER_SECRET``, ``WEIBO_API_KEY``, and/or ``WEIBO_API_SECRET``. For more information and alternative approaches see :doc:`credentials`.
-* Debug mode for sfm-ui is controlled by the ``DEBUG`` environment variable in ``docker-compose.yml``.
-  If setting ``DEBUG`` to false, the ``SFM_HOST`` environment variable must be provided with the host.
-  See the `Django documentation <https://docs.djangoproject.com/en/1.8/ref/settings/#allowed-hosts>`_
-  for ``ALLOWED_HOSTS``.
-* The default timezone is America/New_York (EST/EDT). To select a different timezone, change ``TZ=America/New_York`` in
-  ``docker-compose.yml``.
-* Email is configured by providing the ``SFM_HOST``, ``SFM_SMTP_HOST``, ``SFM_EMAIL_USER``, and ``SFM_EMAIL_PASSWORD``
-  environment variables.  ``SFM_HOST`` is used to determine the host name when constructing links contained in the emails.
-  (If the configured email account is hosted by Google, you will need to configure the account to "Allow less secure apps."
-  Currently this setting is accessed, while logged in to the google account, via https://myaccount.google.com/security#connectedapps).
-
-* The `data volume strategy <https://docs.docker.com/engine/userguide/dockervolumes/#creating-and-mounting-a-data-volume-container>`_
-  is used to manage the volumes that store SFM's data. By default, normal Docker volumes are used; to use
-  a host volume instead, add the host directory to the ``volumes`` field.  This will allow you to access the
-  data outside of Docker.  For example::
-
-    sfmdata:
-        image: ubuntu:14.04
-        command: /bin/true
-        volumes:
-             - /myhost/data:/sfm-data
-
 
 --------------------
  Local installation
 --------------------
 
 Installing locally requires Docker and Docker-Compose. See :ref:`docker-installing`.
 
-1. Either clone this repository::
+1. Either clone this repository and copy the example configuration files::
```

Comment:
I realize this wording precedes the changes here, but how about "the sfm-docker repository" instead of "this repository".